quarkus.ls/com.redhat.quarkus.ls/src/test/java/com/redhat/quarkus/services/QuarkusAssert.java

```diff
@@ -125,14 +125,20 @@ private static void assertCompletion(CompletionList completions, CompletionItem
 		 * expected.kind); }
 		 */
 		// if (expected.getTextEdit() != null && match.getTextEdit() != null) {
-		if (expected.getTextEdit().getNewText() != null) {
-			Assert.assertEquals(expected.getTextEdit().getNewText(), match.getTextEdit().getNewText());
-		}
-		Range r = expected.getTextEdit().getRange();
-		if (r != null && r.getStart() != null && r.getEnd() != null) {
-			Assert.assertEquals(expected.getTextEdit().getRange(), match.getTextEdit().getRange());
+		if (expected.getTextEdit() == null) {

```

Comment:
please don't add this test support. It should be better to write tests every time with text range.